02week/pigLatin.js

```diff
@@ -8,9 +8,56 @@ const rl = readline.createInterface({
 });
 
 
-function pigLatin(word) {
+function pigLatin(input) {
+  let inputStr = input.split('');
+  let counter  = 0;
 
-  // Your code here
+  inputStr.forEach(isVowel);
+  function isVowel(item,index){
+    if(index == 0)
+      console.log('Input letters are:');
+    console.log(item);
+  }
+  let index;
+  let newStr1;
+  let newStr2;
+  let vowelFound = false;
+  for(index = 0; index<inputStr.length;index++){
+    switch(inputStr[index]){
+      case 'a':
```

Comment:
this is a really long switch statement and there's some redundancy, what's a better way to check if something is a vowel?

Reply:
Pushed new changes with refactored code

Reply:
https://github.com/bireswarb/javascript-workbook/pull/7